src/bp-templates/bp-nouveau/includes/groups/ajax.php

```diff
@@ -283,6 +283,10 @@ function bp_nouveau_ajax_joinleave_group() {
 					'type'     => 'success',
 				);
 
+				if ( $group->status == 'hidden' ) {
```

Comment:
@sadrul Use the strict comparision and other looks good to me.
```suggestion
				if (  'hidden' === $group->status ) {
```